FutureFeatures.md

```diff
@@ -561,3 +561,18 @@ static signature validation check. This could be improved by allowing:
   check of a call to a heterogeneous table;
 * any other specific GC reference type, effectively allowing WebAssembly code
   to implement a variety of rooting API schemes.
+
+### Memset and Memcpy Operators
+
+Copying and clearing large memory regions is very common, and making these
+operations fast is architecture dependent. Although this can be done in the MVP
+via `i32.load` and `i32.store`, this requires more bytes of code and forces VMs
+to recognize the loops as well. The following operators can be added to improve
+performance:
+
+* `move_memory`: Copy data from one memory region to another region, copying backward if the regions overlap
+* `zero_memory`: Set all bytes in a memory region to zero
+* `set_memory`: Set all bytes in a memory region to a given byte
```

Comment:
If we have `set_memory` (which I think we should), then I don't think there's much reason for a specialized zero form; we're all just going to be calling `memset()` anyhow.